dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/RegisteredService.java

```diff
@@ -26,9 +22,7 @@
  * {@code RegisteredService.of()}.
  */
 @SuppressWarnings("all")
-@ParametersAreNonnullByDefault
 @Generated({"Immutables.generator", "AbstractRegisteredService"})
```

Comment:
Is this still being generated from Immutables? Were the annotation changes an unexpected side-effect?